packages/cardano-services/test/Program/services/ogmios.test.ts

```diff
@@ -21,13 +21,29 @@ import { SrvRecord } from 'dns';
 import { bufferToHexString } from '@cardano-sdk/util';
 import { clearDbPools, createHealthyMockOgmiosServer, ogmiosServerReady } from '../../util';
 import { createMockOgmiosServer } from '../../../../ogmios/test/mocks/mockOgmiosServer';
+// import { getAliceHandleProviderResponse } from '@cardano-sdk/cardano-services-client';
```

Comment:
```suggestion
// import { getAliceHandleProviderResponse } from '../../../../cardano-services-client/test';
```
Might need to narrow the path further

Reply:
I thought we didn't want to narrow paths.. but in this case, yea, I can do that and then we don't have to move it anywhere.

Reply:
ebc0237c005b9b050f038dcac487ca1e9a5875e6 should solve  this

Reply:
@mkazlauskas please check this and advise. We've have issues with test imports in the past, so just want to ensure we're not tripping over this issue again.

Reply:
I don't know how that even works `'@cardano-sdk/cardano-services-client/test/util';` :thinking: 

I suggest moving `getAliceHandleProviderResponse` to `util-dev`. `'../../../../cardano-services-client/test'` is also acceptable because it's the pattern already in use, but need to be careful with this to make sure that the dependency test project is in `test/tsconfig.ts`